spec/controllers/application_controller/policy_support_spec.rb

```diff
@@ -17,4 +17,28 @@
       controller.send(:assign_policies, Host)
     end
   end
+
+  describe '#policy_sim' do
+    context 'canceling policy simulation' do
+      before do
+        allow(controller).to receive(:flash_to_session).and_call_original
+        allow(controller).to receive(:previous_breadcrumb_url).and_return('')
+        allow(controller).to receive(:redirect_to)
+        controller.instance_variable_set(:@_params, :button => 'cancel')
+      end
+
+      it 'calls redirect_to method' do
+        expect(controller).to receive(:redirect_to).with('')
```

Comment:
Why `with('')`? If you just want to ignore the argument, you can omit the whole method call.

Reply:
I wanted to test if this method was called, no matter the argument

Reply:
then just omit the `with`, no need for it